yt_idv/scene_components/base_component.py

```diff
@@ -30,6 +30,7 @@ class SceneComponent(traitlets.HasTraits):
     use_db = traitlets.Bool(False)  # use depth buffer
     iso_tolerance = traitlets.CFloat(0.025)
     iso_layers = traitlets.List()
+    iso_layers_alpha = traitlets.List()
```

Comment:
This doesn't need to change for this, but I'm genuinely wondering if we should make `iso_layers` a `List` of `Tuple`s.

Reply:
ya, I was kind of thinking that as well! Might make it more flexible for changes down the line too. e.g., adding a "visible" toggle for each layer.

Reply:
So I decided not to make this change here, see my note about a wider re-factor in the latest comment.